lean4-infoview/src/infoview/goals.tsx

```diff
@@ -7,16 +7,21 @@ interface HypProps {
     hyp: InteractiveHypothesisBundle
 }
 
-
 /** Returns true if `h` is inaccessible according to Lean's default name rendering. */
 function isInaccessibleName(h: string): boolean {
     return h.indexOf('✝') >= 0;
 }
 
-export function Hyp({ hyp : h }: HypProps) {
+export function Hyp({ hyp: h }: HypProps) {
+    let namecls : string = ''
+    if (h.isInserted) {
+        namecls += 'inserted-text '
+    } else if (h.isRemoved) {
+        namecls += 'removed-text '
+    }
     const names = InteractiveHypothesisBundle_accessibleNames(h).map((n, i) =>
-            <span className={'mr1 ' + (isInaccessibleName(n) ? 'goal-inaccessible' : '')} key={i}>{n}</span>
-        )
+        <span className={ 'mr1 ' + (isInaccessibleName(n) ? 'goal-inaccessible ' : '') + namecls} key={i}>{n}</span>
```

Comment:
Minor, but you could default to `namecls : string = 'mr1 '` to shorten this line a bit.